### PR TITLE
Code clean up

### PR DESCRIPTION
**Pull request by chriskinch**

### CODE REVIEW

File: src/Asset/CssCollectionGrouper.php [hunk 1]
```diff
@@ -16,9 +21,11 @@ public function group(array $css_assets) {
     // Alter inline group so that it isn't aggregated with regular files.
     foreach ($css_assets as &$item) {
       if (!empty($item['inline'])) {
-        $item['group'] = $item['group'] - 1;
```
Comment (davidgrayston) [inline src/Asset/CssCollectionGrouper.php:19]: @chriskinch I tried to keep the SMACSS grouping here by subtracting 1. Having one constant for all inline style means that we can't categorise the style elements (which may be what we want?). Which approach do you think is better?

Comment (chriskinch) [reply src/Asset/CssCollectionGrouper.php:19]: Good point didn't think about this. Seems to work better your way!

Comment (chriskinch) [reply src/Asset/CssCollectionGrouper.php:19]: Actually, on closer inspection I can't see that it makes any difference either way. Tried it with multiple CSS assets in the same theme set as inline and it still respects the order in the YML. If 2 assets are one after each other they bundle together which is good.

Slightly unexpected but I'll take it!

File: src/Asset/CssCollectionRenderer.php [hunk 2]
```diff
@@ -4,30 +4,41 @@
...
+
     foreach ($css_assets as $css_asset) {
       if (!empty($css_asset['inline'])) {
         // Render inline element.
-        $element = [
-          '#type' => 'html_tag',
-          '#tag' => 'style',
-          '#value' => $this->getCssFileContents($css_asset['data']),
-          '#attributes' => [
-            'type' => 'text/css',
```
Comment (davidgrayston) [inline src/Asset/CssCollectionRenderer.php:22]: @chriskinch is type not required as it defaults to text/css?

File: src/Asset/CssCollectionGrouper.php (hunk 1 above)
Comment (davidgrayston) [reply src/Asset/CssCollectionGrouper.php:19]: Yes, seems to make no difference as you say

**Pull request merged**